core/jobs_registry.py

```diff
@@ -226,6 +226,10 @@ class ContinuousComputationEventDispatcher(python_utils.OBJECT):
     def dispatch_event(cls, event_type, *args, **kwargs):
         """Dispatches an incoming event to the ContinuousComputation
         classes which listen to events of that type.
+
+        Args : str . Type of event.
+
+
```

Comment:
There should be no spaces before the closing of doc string.

Reply:
I will resolve that .